1,3,4.TXT

```diff
@@ -0,0 +1,7 @@
+--1, 3, 4 tasks;
+  CREATE TABLE folks (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  first_name VARCHAR NOT NULL UNIQUE,

```

Comment:
This UNIQUE constraint doesn't allow to add unique names for all records and not only for pair `first_name` `last_name`. For example, if there is a record `John Snow` it is impossible to add `John Walker`, but it should be possible since these ara different names

Reply:
Got it